extensions/transferto/action.go

```diff
@@ -69,7 +69,7 @@ func (a *TransferAirtimeAction) Execute(run flows.FlowRun, step flows.Step) erro
 		run.LogEvent(step, events.NewErrorEvent(fmt.Errorf("can't transfer airtime to contact without a tel URN")))
 		return nil
 	}
-	recipient := telURNs[0].Path()
+	recipient := telURNs[0].URN().Path()
```

Comment:
`ContactURN` used to be a `urns.URN` but now the urn part is a field so we can update it when `ContactURN.SetChannel` is called